actor-model/README.md

```diff
@@ -0,0 +1,52 @@
+# Actor Model
```

Comment:
Check out the `README.md` template here: https://github.com/iluwatar/java-design-patterns/wiki/21.-New-Pattern-Template